hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/AbstractOMKeyDeleteResponse.java

```diff
@@ -115,15 +114,15 @@ protected void addDeletionToBatch(
    *  file table (which is in prefix format) and adds the fullKey
    *  into the deletedTable
    * @param keyName     (format: objectId/key)
-   * @param fullKeyName (format: vol/buck/key)
+   * @param deleteKeyName (format: <keyname>/keyObjectId)
    * @param omKeyInfo
    * @throws IOException
    */
   protected void addDeletionToBatch(
       OMMetadataManager omMetadataManager,
       BatchOperation batchOperation,
       Table<String, ?> fromTable,
-      String keyName, String fullKeyName,
```

Comment:
The `fullKeyName` passed here is different still `/volumeName/bucketName/keyName/objectId` please check in other places as well.

Reply:
@aswinshakil Its verified and handled to have /volumeName/bucketName/keyName/objectId, Its done as part of below PR splitted from this, HDDS-8462